firebase-crashlytics/ktx/src/main/kotlin/com/google/firebase/crashlytics/ktx/FirebaseCrashlytics.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.ktx
+
+import androidx.annotation.Keep
+import com.google.firebase.FirebaseApp
+import com.google.firebase.components.Component
+import com.google.firebase.components.ComponentRegistrar
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.platforminfo.LibraryVersionComponent
+
+/** Returns the [FirebaseCrashlytics] instance of the default [FirebaseApp]. */
+val Firebase.crashlytics: FirebaseCrashlytics
+    get() = FirebaseCrashlytics.getInstance()
+
+fun FirebaseCrashlytics.setCustomKeys(vararg pairs: Pair<String, Any>) {
```

Comment:
This is definitely an improvement compared to using plain java api, a few considerations though:
* Some other ktx sdks use dsl-like notation for similar use-cases, e.g. [Firebase.analytics.logEvent](https://firebase.google.com/docs/reference/kotlin/com/google/firebase/analytics/ktx/package-summary?hl=en#logevent), so it might make sense to something similar here for the sake of consistency. The dsl could even support `infix` notation like `Pair` does(not sure if it would be a good idea though).
* The dsl would also be more type-safe, i.e. there won't be a need for `IllegalArgumentException` below, as the block can be made to only have the methods with argument types crashlytics supports.

So I would suggest something along the lines of:
```kotlin
crashlytics.setCustomKeys {
  key("foo", "bar")
  key("int", 42)
  // or
  "foo" to "bar" // this might not be a good idea as it shadows the Pair constructor function.
}
```

wdyt?

Reply:
@vkryachko since `analytics.logEvent` uses DSL, I think we should maintain consistency. Ive updated the code to match your suggestion. PTAL